torch/_tensor_docs.py

```diff
@@ -2384,6 +2384,14 @@ def callable(a, b) -> number
     0.012766935862600803
 """)
 
+add_docstr_all('to_sparse',
```

Comment:
This requires an entry in `docs/source/sparse.rst` to be visible.

Reply:
I built the docs locally, added ```    .. method:: to_sparse ``` in that file, but still wasn't able to get this method to show up in the docs.  Is that what you meant to do?

Reply:
Oh, my bad. It is supposed to go into `docs/source/tensors.rst`.

Reply:
got it, should be good now